code/datums/components/attachment_handler.dm

```diff
@@ -366,7 +366,7 @@
 			else
 				icon = attachment_data[OVERLAY_ICON]
 				suffix = attachment.icon == icon ? "_a" : ""
-		var/mutable_appearance/new_overlay = mutable_appearance(icon, icon_state + suffix, -attachment_data[ATTACHMENT_LAYER])
+		var/mutable_appearance/new_overlay = mutable_appearance(icon, icon_state + suffix, attachment_data[ATTACHMENT_LAYER])
```

Comment:
I assume this was negative to use float layers or something? issue is this seems to not have been changing the actual in-world appearance, and the dummy copy was inverting rendering attachments (i.e visor then human then helmet, instead of human then visor then helmet) didnt see anything broken from thsi change but this is why it should be tmed a little

Reply:
Weird. I remember something using float layers because it needed to, but can't think off hand of why attachments would.
Looking at the history, it may just be a type, as Hyper added it when he made some changes for chameleon capes.